sharc/antenna/antenna_rs1813.py

```diff
@@ -65,16 +65,16 @@ def calculate_gain(self, *args, **kwargs) -> np.array:
     # initialize antenna parameters
     param = ParametersEessSS()
     param.antenna_pattern = "ITU-R RS.1813-1"
-    param.frequency = 23900
-    param.antenna_gain = 52
-    param.antenna_diameter = 2.2
-    param.antenna_efficiency = 0.6
+    param.frequency = 10650
+    param.antenna_gain = 39.6
+    param.antenna_diameter = 1.1
+    param.antenna_efficiency = 0.606
     antenna = AntennaRS1813(param)
 
     gain = antenna.calculate_gain(off_axis_angle_vec = phi)
 
     fig = plt.figure(figsize=(8,7), facecolor='w', edgecolor='k')  # create a figure object
-    plt.semilogx(phi, gain - param.antenna_gain, "-b", label = "$f = 23.9$ GHz")
+    plt.semilogx(phi, gain, "-b", label = "$f = 10.650$ GHz")
```

Comment:
Eu achei estranho o plot dessa antena. Confirma que é isso? Se for é melhor aumentar o limite do eixo y.

![image](https://github.com/user-attachments/assets/e1a3b1aa-ec98-4ae8-a600-212292df786f)

Reply:
Não era nem pra eu ter commitado as mudanças que fiz nesse arquivo. Provavelmente commitei incompleto no meio do caminho, e como sabia que não era pra alterar esse arquivo, não commitei alterações nele depois.

Mas, pra não deixar de responder sua pergunta, era assim mesmo que usei o gráfico, só tinha mudado o limite do eixo y